### PR TITLE
#227: Transitioned ProblemHeader to react component

### PR DESCRIPTION
**Pull request by oskarhinc**

### CODE REVIEW

File: src/components/Editor/components/ProblemHeader/index.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        return (
+            <div className={problem.header}>
+                <div>
+                    <span id="ProblemTitle" className={problem.title} role="heading" aria-level="1">Problem title</span>
+                    <span id="ProblemMath" className={problem.title}>Problem</span>
+                </div>
+                <div className={problem.btnContainer}>
+                    <button aria-hidden='true'
+                        //TODO: onclick="introJs().setOption('tooltipClass', 'introjs-helperLayer').start(); GoogleAnalytics('Tour');"
+                        className={classNames({[bootstrap.btn]: true, [styles.pointer]: true,
```
Comment (pawelcieszko) [inline src/components/Editor/components/ProblemHeader/index.js]: use identation like JSON instead of inline style

File: src/styles/styles.css [hunk 2]
```diff
@@ -7,3 +7,7 @@
     width: 1px;
     overflow: hidden;
   }
+
+.pointer {
+    cursor:pointer;
```
Comment (dserkowski) [inline src/styles/styles.css]: missing the space character between 'cursor:' and 'pointer'

Comment (pawelcieszko) [inline src/styles/styles.css]: add whitespace

File: src/index.html [hunk 3]
```diff
@@ -100,18 +100,7 @@ <h2>Select a problem and try out Benetech's Math Editor (alpha)!</h2>
             <div id="ProblemList"></div> <!-- place for list of problems -->
         </nav>
         <main id="MainWorkArea" class="editorAndHistoryWrapper">
-            <div class="problemHeader">
-                <div>
-                    <span class="problemTitle" id="ProblemTitle" role="heading" aria-level="1">Problem title</span>
-                    <span class="problemTitle" id="ProblemMath">Problem</span>
```
Comment (afijal) [inline src/index.html]: Maybe we could pass an equation here as an parameter? What do you think?
https://reactjs.org/docs/components-and-props.html

Comment (oskarhinc) [reply src/index.html]: Thanks, I'll try to do this in a new PR.

**Pull request merged**